tutorial/callback-example/package.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "getProductById-example",
+  "version": "0.0.1",
+  "main": "getProductById.js",
+  "description": "An example getProductById implementation.",
+  "private": true,
+  "author": {
+    "name": "Adobe Systems Inc."
+  },
+  "dependencies": {
+    "node-rest-client": "3.1.0"
+  },
+  "scripts": {
+    "zip": "rm getProductById.zip; zip -q -r getProductById.zip getProductById.js package.json node_modules",
```

When this is run for the first time and `getProductById.zip` does not exist, it will print that there is no such file or directory. It does not impact the behavior but it looks like an error.

We can add `if [ -f getProductById.zip ]; then rm getProductById.zip; fi` to only delete the file if it exists.

---

actually I just found out that the `-f` option for `rm` also means to silently fail if a file doesn't exist.